.github/workflows/main.yml

```diff
@@ -2,37 +2,28 @@
 name: "Main Workflow"
 env:
   DOCKER_USERNAME: sharifwss
-  RELEASE_VERSION: 1.0.0
+  RELEASE_VERSION: 1.0.1
   TARGET_HOST: 37.152.187.105
   HOST_SSH_USER: wss
-on:
-  push:
-    branches:
-      - master
+ on:
+   push:
+     branches:
+       - master
```

Comment:
What's this on-space indentation for?

Reply:
Oh, they were left unintentionally. Removed.